src/services/validatorSchema.ts

```diff
@@ -0,0 +1,6 @@
+import * as yup from 'yup';
+
+export const inventoryCreateSchema = yup.object({
+    user_id: yup.string().defined("All fields must be completed"),
```

Comment:
same here, let's have an explicit error, for instance "user_id must be provided"